plenum/common/batched.py

```diff
@@ -133,7 +134,9 @@ def flushOutBoxes(self) -> None:
 
         for rid in removedRemotes:
             logger.warning("{}{} has removed rid {}"
-                           .format(CONNECTION_PREFIX, self, rid),
+                           .format(CONNECTION_PREFIX, self,
+                                   rid if isinstance(rid, str) else
```

Comment:
Please move `rid if isinstance(rid, str) else ...` to helper method as well

Reply:
ok